brainscore/submission/configuration.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+import distutils
+
+from brainscore.submission.models import Model, Submission
+
+
+class BaseConfig:
+    def __init__(self, work_dir, jenkins_id, db_secret, config_path):
+        self.work_dir = work_dir
+        self.jenkins_id = jenkins_id
+        self.config_path = config_path
+        self.db_secret = db_secret
+        return
+
+
+class MultiConfig(BaseConfig):
```

Comment:
Some documentation/comments here would be helpful to understand what the difference between `MultiConfig` and `SubmissionConfig` is and when to use which one